include/seastar/http/routes.hh

```diff
@@ -129,6 +137,8 @@ public:
      * Example  routes.add_default_handler(handler);
      * @param handler
      * @return
+     * @attention This method does not take ownership of the handler's pointer. It is
+     * the callee responsibility to delete it
```

Comment:
> It is the callee responsibility to delete it

should replace "callee" with "caller". but i think this line is redundant. we are not obliged to encourage to programmer to write leak-free code. but we should be very explicit on the ownership and the expectation of the life cycle, if the ownership is not transferred to us.

so i'd suggest replacing this line with

> The caller must keep the @c handler active as long as the route instance is active.

Reply:
done